docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml

```diff
@@ -62,11 +76,53 @@ code: |
 include:
   - juvenile-sealing-10.yml
 ---
+comment: |
+  if click on docket sheets link
+    take to docket sheets flow
+  if click others
+    if haven't seen docket sheets link
+      take to docket sheets flow
+    else
+      go to next question...?
+
+  Clicking link triggers
+    new code block that
+      undefines values
+      triggers `url_action()`
+---
+id: tangent_start_state
```

Comment:
Is this ever called?

Reply:
I think it might be called by the popover snippet.